### ISSUE TITLE
Not clear error for not-verified users

### ISSUE THREAD

**Issue by ppitonak** [stormpath/express-stormpath#242]
consider this scenario:

1. user registers using username/email and password
2. he ignores the email with verification link
3. after some time tries to log in

Login form shown 'Invalid username or password' error even though his username and password are correct but the email hasn't been verified yet.

**Comment by robertjd** [comment]
Yes, this is the current behavior and it's not great.  Until we can amend this to give you more control, I would suggest implementing a custom version of our login template and having it respond to actual error code from stormpath (which does indicate what is actually happening).  Please let user know if we can offer assistance with this.
